eth/downloader/downloader.go

```diff
@@ -1201,3 +1202,7 @@ func (d *Downloader) deliver(destCh chan dataPack, packet dataPack, inMeter, dro
 		return errNoSyncActive
 	}
 }
+
+func (d *Downloader) DropPeer(peer *eth.Peer) {
```

Comment:
I created this method as a quick fix. Can we see if returning error drops the peer?

Reply:
its not the same, dropPeer is preferred in this case